src/pages/lawfirm/(dashboard)/ClaimPredictor.jsx

```
@@ -90,56 +86,54 @@ export default function ClaimPredictor() {
 
                 {result?.case_advice && (
                   <div className="mt-6 p-8 bg-white shadow-lg rounded-2xl">
                     <h2 className="text-2xl font-bold" style={{ color: "#343434", fontFamily: "Poppins, sans-serif", fontSize: "90%" }}>Case Advice</h2>
-                    <p
-                      className="mt-2 text-lg leading-relaxed whitespace-pre-line"
-                      dangerouslySetInnerHTML={{
-                        __html: result.case_advice.replace(/\*\*(.*?)\*\*/g, "<strong>$1</strong>"),
-                      }}
-                      style={{ color: "#343434", fontFamily: "Poppins, sans-serif", fontSize: "90%" }}
-                    ></p>
+                    <p className="mt-2 text-lg leading-relaxed whitespace-pre-line" dangerouslySetInnerHTML={{ __html: result.case_advice.replace(/\*\*(.*?)\*\*/g, "<strong>$1</strong>") }} style={{ color: "#343434", fontFamily: "Poppins, sans-serif", fontSize: "90%" }}></p>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)